src/App.test.jsx

```diff
@@ -0,0 +1,81 @@
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ListProvider from './context/ListContext';
+import App from './App';
+
+beforeEach(() => {
+  render(
+    <ListProvider>
+      <App />
+    </ListProvider>
+  );
+});
+
+test('renders page header', () => {
```

Comment:
We could potentially combine some of these tests into one large test that tests the entire functionality your app. 
https://kentcdodds.com/blog/write-fewer-longer-tests